html.go

```diff
@@ -124,7 +124,7 @@ func (w *HTMLWriter) Newline() {
 
 // NewHTMLRenderer creates and configures an HTMLRenderer object, which
 // satisfies the Renderer interface.
-func NewHTMLRenderer(params HTMLRendererParameters) Renderer {
+func NewHTMLRenderer(params HTMLRendererParameters) *HTMLRenderer {
```

Comment:
Latex renderer is currently a bit of an orphaned child. It was decided that different renderers are not a priority for Blackfriday, up to the point where people are welcome to extract it to a separate repo and maintain it there.